docs/_docs/04_community/contributing.md

```diff
@@ -184,6 +189,8 @@ We have a separate CI pipeline for Azure code. To run it on a forked repo:
 
 1. Wait for the `ci-workflow` to be finished
 
+    > The pipeline will use the given Azure subscription and create resources on it. When the test finish, resources will be tear-down. Creating resources may cost to the Azure subscription, though. We didn't expect a left-over resource, but please keep in mind that, sometimes due to a bug or a glitch, some resources may be left behind.

```

Comment:
```suggestion
    > The pipeline will use the given Azure subscription and create resources as part of running the test in this subscription.  When the test finish the resources are torn down. You will incur some cost for these resources during the execution.  Normal operation should not leave any resource post CI execution, but it can happen.  You should verify that the resources have been deleted post workflow execution.

```

Reply:
contributing guideline updated, thanks for the feedback